Scope.Client/Loader/Loader.cs

```diff
@@ -65,21 +157,21 @@ public static void LoadAll()
         /// </summary>
         public static void EnableMods()
         {
-            List<IMod<IConfig>> toLoad = Mods;
+            List<IMod<IConfig>> toLoad = Mods.ToList();
 
             foreach (IMod<IConfig> mod in toLoad)
             {
                 try
                 {
-                    if (mod.Name.StartsWith("Scope") && mod.Config.IsEnabled)
-                    {
-                        mod.OnEnabled();
-                        toLoad.Remove(mod);
-                    }
+                    if (!mod.Name.StartsWith("Scope") || !mod.Config.IsEnabled)
```

Comment:
Why not load mods that names don’t start with Scope? Unless if it’s for compatibility or something

Reply:
It's not how this logic works, don't worry, it's fine.